### PR TITLE
Adding a mirror node update section to HIP-869

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:
This PR details the changes the mirror node will make to support HIP-869.

**Related issue(s)**:
Fixes #

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by steven-sheehy — APPROVED** [review]

**Review by jsync-swirlds — COMMENTED** [review]
One item to note; not sure if it makes a large difference.

File: HIP/hip-869.md [hunk 1]
```diff
@@ -563,6 +563,58 @@ When executing the next `freeze` transaction with `freeze_type` set to `PREPARE_
...
+The following changes will be made to the REST API:
+- Update the `/api/v1/network/nodes` endpoint to return `domain_name` and `admin_key`
+
+Example response:
+
+```json
+{
+  "nodes": [
+    {
+      "admin_key": "0x565fd764f0957fa170a676210c9bdbddf3bc9519702cf927fa6767a40463b96f",
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply HIP/hip-869.md:581]: True, actually this syntax is wrong and it should be a wrapper object with a `_type` field as is our standard in the REST API.

[the remark this answers is not in the document]
Comment (mgoelswirlds) [reply HIP/hip-869.md:581]: Ok will update this.

[the remark this answers is not in the document]
Comment (mgoelswirlds) [reply HIP/hip-869.md]: Done.

**Review by jsync-swirlds — APPROVED** [review]

**Review by jsync-swirlds — APPROVED** [review]

**Review by mgarbs — APPROVED** [review]

**Pull request merged**